assets/shaders/glsl/textured_line_b_v.glsl

```diff
@@ -42,7 +42,7 @@ vec4 globe_coords(vec2 world_pos) {
 	new_world_pos.xyz += 0.5; 	// Move the globe to the center
 
 	return vec4(
-		(2. * new_world_pos.x - 1.f)  * zoom,
```

Comment:
Does putting the aspect ratio adjustment here work properly? It was moved out originally to fix the joins where the lines met doing weird things.

Reply:
I haven't noticed. How did weirdness looked like?

Reply:
You might have to zoom in pretty closely to see it. The problem was that when lines made sharper turns you would sometimes see the corners "stick out" too much. It was caused by the lines needing to also project the neighboring points in order to know the angle between them, which in turn determines where the points for the join need to go. Scaling for the aspect ratio threw off those calculations.

Reply:
This shader behavior should be fixed in the next commit.